third_party/terraform/tests/resource_bigquery_data_transfer_config_test.go

```diff
@@ -164,6 +164,10 @@ resource "google_bigquery_dataset" "my_dataset" {
   location      = "asia-northeast1"
 }
 
+resource "google_pubsub_topic" "my_topic" {
+  name = "my_topic"
```

Comment:
Can this be given a name with randomness like we do for the dataset? That'll make sure separate test runs don't conflict. Also starting it with `tf-test` is a good practice because we have some automation that cleans up resources in our test environment with that prefix.